src/lib/blueprint.ts

```diff
@@ -826,12 +827,17 @@ const createResource = (
   schemaName: string,
   schema: OpenapiSchema,
 ): Resource => {
+  const routePath = schema['x-route-path']
+  if (routePath == null || routePath.length === 0) {
+    throw new Error(`Missing route path for ${schemaName}`)
+  }
```

Comment:
We should also validate that the route path actually exists in routes.

Reply:
@razor-x what should we do about /health routePath defined on [`service_health` resource](https://github.com/seamapi/seam-connect/blob/a0230f17f9740dd7a799eff40bd3ff9612d21931/lib/zod/health.ts#L13)? `/health/*` endpoints are excluded from openapi spec so no route is generated to validate `service_health`'s routePath against.

![image](https://github.com/user-attachments/assets/91653fc4-d226-4d5e-b88b-be6b02bde7b3)

Reply:
@razor-x did you have a chance to take a look at this?

Reply:
We should remove service_health as a resource then.